metadata-ingestion/src/datahub/ingestion/source/ldap.py

```diff
@@ -117,12 +140,9 @@ class LDAPSourceConfig(ConfigModel):
 
 
 @dataclasses.dataclass
-class LDAPSourceReport(SourceReport):
-
-    dropped_dns: List[str] = dataclasses.field(default_factory=list)
-
+class LDAPSourceReport(StaleEntityRemovalSourceReport):
     def report_dropped(self, dn: str) -> None:
-        self.dropped_dns.append(dn)
+        self.report_stale_entity_soft_deleted(dn)
```

Comment:
Revert this change. Stale entity soft deletion is automatically reported from [here](https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/source/state/stale_entity_removal_handler.py#L225)

Reply:
competed